src/schema.toml

```diff
@@ -25,6 +25,9 @@ features.show_avatar_pipelines_link = { category = "features", type = "boolean",
 features.show_model_collection_link = { category = "features", type = "boolean", name = "Show Model Collection Link", description = "Show link to a collection of models." }
 features.hide_powered_by = { category = "features", type = "boolean", name = "Hide Powered By Hubs", description = "Hides the 'Powered by Hubs Cloud' notice on the home page and loading screens." }
 
+features.public_rooms = { category = "features", type = "boolean", name = "Public Rooms", description = "Allow non-admin users to create public rooms that will be listed on the homepage." }
```

Comment:
I'd change this to `Allow visitors to set rooms to public to feature them on the homepage.`